### PR TITLE
Edits styling for poster container

### PR DESCRIPTION
**Pull request by sted1994**
## What does this PR do?
Fixed poster alignment in poster container

## Where should the reviewer start?
poster-container.js

## How should this be manually tested?
Run npm test

### CODE REVIEW

File: src/css/poster-container.css [hunk 1]
```diff
@@ -1,5 +1,7 @@
 .poster-container {
-  display: grid;
+  display: flex;
```
Comment (emmacbd) [inline src/css/poster-container.css:2]: Thanks for fixing this.

**Review by emmacbd — COMMENTED** [review]
Looks good, thanks!

File: src/css/poster-container.css [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+  flex-wrap: wrap;
+  justify-content: center;
   grid-template-columns: repeat(4, 1fr);
```
Comment (michaelputnam67) [inline src/css/poster-container.css:5]: dont think we need this anymore

**Review by michaelputnam67 — COMMENTED** [review]
Looks good! just some minor refactoring later on.

**Pull request merged**